src/js/createCase.js

```diff
@@ -0,0 +1,84 @@
+import Cookies from 'js-cookie';
+import * as Sentry from '@sentry/browser';
+import logger from './jwt/logger';
+const log = logger('createCase.js');
+import { spinUpStore } from './redux-config';
+
+// Lit of products that are bundles
+const BUNDLE_PRODUCTS = [
+    'Red Hat OpenShift Cluster Manager',
+    'Red Hat Insights',
+    'Subscription Watch'
+];
+
+// List of products that are individual apps
+const APP_PRODUCTS = [
+    { id: 'automation-hub', name: 'Ansible Automation Hub' },
+    { id: 'automation-analytics', name: 'Ansible Automation Analytics' },
+    { id: 'migrations', name: 'Red Hat Migration Analytics' },
```

Comment:
@ryelo I think this one should be just `Migration Analytics`, I don't see `Red Hat Migration Analytics` in the product dropdown.